mlir/test/Dialect/XeGPU/xegpu-wg-to-sg-rr.mlir

```diff
@@ -7,15 +7,29 @@ gpu.module @test_round_robin_assignment {
       // CHECK-COUNT-4: xegpu.create_nd_tdesc %[[ARG_0]][%{{.*}}, %{{.*}}] : memref<256x128xf32>
       // CHECK-SAME: -> !xegpu.tensor_desc<16x16xf32, #xegpu.layout<lane_layout = [1, 16], lane_data = [1, 1]>>
       // CHECK-NOT: xegpu.create_nd_tdesc
-      %tdesc = xegpu.create_nd_tdesc %src[0, 0] : memref<256x128xf32>
+      %cst0 = arith.constant 0 : index
```

Comment:
I think it is simpler to keep a separate copy of test file, which focus on load_nd/store_nd/prefetch_nd+offset. When transition is over, we simply remove the current test file.